### PR TITLE
Typo "Teams Room"→"Teams Rooms"

### PR DESCRIPTION
**Pull request by hyoshioka0128**
https://learn.microsoft.com/en-us/microsoftteams/hybrid-meetings-device-select?tabs=signature 
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/hybrid-meetings-device-select.md 
#PingMSFTDocs

### CODE REVIEW

File: Teams/hybrid-meetings-device-select.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ If you want more flexibility than a sound bar provides, you can choose cameras,
 
 ## Touchscreen console and compute module
 
-The touchscreen console is the device that users interact with when joining and managing a meeting. The compute module is the brains of the Teams Room and runs Windows 10 Enterprise IoT.
+The touchscreen console is the device that users interact with when joining and managing a meeting. The compute module is the brains of the Teams Rooms and runs Windows 10 Enterprise IoT.
```
Comment (JohanFreelancer9) [inline Teams/hybrid-meetings-device-select.md:44]:
```suggestion
The touchscreen console is the device that users interact with when joining and managing a meeting. The compute module is the brains of Teams Rooms and runs Windows 10 Enterprise IoT.
```

File: Teams/hybrid-meetings-device-select.md [hunk 2]
```diff
@@ -164,7 +164,7 @@ The following are examples of audio systems for Teams Rooms:
 
 ## Signature Teams Rooms touchscreen consoles and compute modules
 
-The touchscreen console is the device that users interact with when joining and managing a meeting. The compute module is the brains of the Teams Room and runs Windows 10 Enterprise IoT.
+The touchscreen console is the device that users interact with when joining and managing a meeting. The compute module is the brains of the Teams Rooms and runs Windows 10 Enterprise IoT.
```
Comment (JohanFreelancer9) [inline Teams/hybrid-meetings-device-select.md:167]:
```suggestion
The touchscreen console is the device that users interact with when joining and managing a meeting. The compute module is the brains of Teams Rooms and runs Windows 10 Enterprise IoT.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @hyoshioka0128, kindly commit the suggested changes. Many thanks.

**Comment by hyoshioka0128** [comment]
@JohanFreelancer9 - Thanks for the suggestion.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**